### PR TITLE
feat: Upgrade to CUDA 12.5

### PR DESCRIPTION
**Pull request by reneleonhardt**
**Description**

### Features
* Upgrade to CUDA 12.5
* Add Docker tag suffix `-cuda11`

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/image_build.yml [hunk 1]
```diff
@@ -274,6 +278,33 @@ jobs:
         run: |
           echo "Image is available at ttl.sh/localai-ci-pr-${{ github.event.number }}:${{ steps.meta_pull_request.outputs.version }}" >> $GITHUB_STEP_SUMMARY
 ## End testing image
+      - name: Build and push CUDA 11
```
Comment (mudler) [inline .github/workflows/image_build.yml:281]: We already push cuda-11 images as part of the build matrix here https://github.com/mudler/LocalAI/blob/06de54203206577fb8cfa779be11ba880d7430c0/.github/workflows/image.yml#L65 . I'm not sure if there is anything to add here, except changing image's defaults.

Comment (reneleonhardt) [reply .github/workflows/image_build.yml:281]: Good catch, I'll remove it.

Comment (reneleonhardt) [reply .github/workflows/image_build.yml:281]: Makefile `docker-aio-all:` looks strange, a duplicated command, should that be removed or `DOCKER_AIO_SIZE` changed in the second call?

Comment (mudler) [reply .github/workflows/image_build.yml]: Good catch - that looks a leftover, should be dropped as its not used anymore! Thanks!

**Review by mudler — APPROVED** [review]

**Comment by mudler** [comment]
Looking good! thank you @reneleonhardt !

**Comment by reneleonhardt** [comment]
@mudler FYI Nvidia seems to support Ubuntu LTS from April now:
https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2404/x86_64/
Maybe check your Docker images for security fixes, then CUDA 12.5 could be built with a new base image.

**Pull request merged**